Core/HLE/sceNetAdhoc.cpp

```diff
@@ -885,15 +1065,23 @@ static int sceNetAdhocctlScan() {
 				return ERROR_NET_ADHOCCTL_DISCONNECTED; // ERROR_NET_ADHOCCTL_BUSY 
 			}
 
+			// Does Connected Event's mipscall need be executed after returning from sceNetAdhocctlScan ?
```

Comment:
yep we can only do scan before updating Adhocctlhandler into connected state , scanning on CONNECT STATE lead to crash on real psp